### DISCUSSION TITLE
Support for Bazel rules_oci

### DISCUSSION THREAD

**Discussion by malt3** [renovatebot/renovate#21066] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Renovate already has excellent support for many types of Bazel WORKSPACE rules, including `rules_docker` (with the `container_pull` rule).
[`rules_oci`](https://github.com/bazel-contrib/rules_oci) is a fresh implementation of the container image handling inside bazel that is similar to rules_docker.
It has a WORKSPACE rule [`oci_pull`](https://github.com/bazel-contrib/rules_oci/blob/main/docs/pull.md) that is very similar to `container_pull`, with the only difference (for renovate) being that the `registry` and `repository` attributes of the rule are combined as `image` in `oci_pull`.

I'd like to upgrade `oci_pull` rules using renovate and would like to know if this is something that could be implemented (by the community, the renovate team or me)?

**Comment by rarkins** [comment]
This sounds like a good idea. I'm assuming that the existing "docker" datasource can be reused as some other managers already do similar.

None of the regular maintainers of Renovate use Bazel in our work, so it would likely need a community contribution from yourself or others. If you're not ready to start then a feature request issue with accompanying reproduction repo would still be good.

**Comment by malt3** [comment]
Resolved by #21216
